packages/twenty-docs/docs/start/local-setup.mdx

```diff
@@ -0,0 +1,226 @@
+If you haven't already set up SSH keys, you can learn how to do so [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/about-ssh). 
+```bash
+git clone git@github.com:twentyhq/twenty.git
+```
+</TabItem>
+<TabItem value="https" label="HTTPS" >
+
+```bash
+git clone https://github.com/twentyhq/twenty.git
+```
+
+</TabItem>
+</Tabs>
+
+## Step 2: Position yourself at the root
+
+```bash
+cd twenty
+```
+
+You should run all commands in the following steps from the root of the project.
+
+## Step 3: Set up a PostgreSQL Database
+We rely on [pg_graphql](https://github.com/supabase/pg_graphql) and recommend you use the scripts below to provision a database with the right extensions.  
+You can access the database at [localhost:5432](localhost:5432), with user `twenty` and password `twenty` .
+
+<Tabs>
+  <TabItem value="linux" label="Linux" default>
+    <b>Option 1:</b> To provision your database locally:
+    ```bash
+    make postgres-on-linux
+    ```
+
+    <b>Option 2:</b> If you have docker installed:
+    ```bash
+    make postgres-on-docker
+    ```
+  </TabItem>
+  <TabItem value="mac-os" label="Mac OS" default>
+    <b>Option 1:</b> To provision your database locally with `brew`:
+    ```bash
+    make postgres-on-macos-intel #for intel architecture
+    make postgres-on-macos-arm #for M1/M2/M3 architecture
+    ```
+
+    <b>Option 2:</b> If you have docker installed:
+    ```bash
+    make postgres-on-docker
+    ```
+  </TabItem>
+  <TabItem value="wsl" label="Windows (WSL)">
+    <b>Option 1 :</b> To provision your database locally:
+    ```bash
+    make postgres-on-linux
+    ```
+
+    Note: you might need to run `sudo apt-get install build-essential` before running the above command if you don't have the build tools installed.
+
+    <b>Option 2:</b> If you have docker installed:  
+    Running Docker on WSL adds an extra layer of complexity.
+    Only use this option if you are comfortable with the extra steps involved, including turning on [Docker Desktop WSL2](https://docs.docker.com/desktop/wsl).
+    ```bash
+    make postgres-on-docker
+    ```
+  </TabItem>
+</Tabs>
+
+
+## Step 4: Setup environment variables
+
+Use environment variables or `.env` files to configure your project.
+
+Copy the `.env.example` files in `/front` and `/server`:
+```bash
+cp ./packages/twenty-front/.env.example ./packages/twenty-front/.env
+cp ./packages/twenty-server/.env.example ./packages/twenty-server/.env
+```
+
+## Step 5: Installing dependencies
+
+:::info
+
+Use `nvm` to install the correct `node` version. The `.nvmrc` ensures all contributors use the same version.
+
+:::
+
+To build Twenty server and seed some data into your database, run the following commands:
+```bash
+nvm install # installs recommended node version
+nvm use # use recommended node version
+
+yarn
+```
+
+## Step 6: Running the project
+
+Setup your database with the following command:
+```bash
+npx nx database:reset twenty-server
```

Comment:
All commands should be: npx nx run my-package:my-command
Here: npx nx run twenty-server:database:reset